main/CommandHandler.cpp

```diff
@@ -256,6 +256,24 @@ int getTemperature(const uint8_t command[], uint8_t response[])
   return 9;
 }
 
+int getDnsIP(const uint8_t command[], uint8_t response[])
```

Comment:
Since we have a matching `setDNSconfig`...

```suggestion
int getDNSconfig(const uint8_t command[], uint8_t response[])
```

Reply:
the name is 'standard' in esp8266, esp32, WiFiS3 and in my WiFiEspAT now too